website/example/Textarea.demo.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Textarea from '@igloo-ui/Textarea';
```

Comment:
It creates an error during the build
```suggestion
import Textarea from '@igloo-ui/textarea';
```